custom/icds_reports/management/commands/queue_person_cases.py

```diff
@@ -0,0 +1,30 @@
+
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.models import AsyncIndicator
+from django.db import connections
+
+from dimagi.utils.chunked import chunked
+
+
+class Command(BaseCommand):
+    help = "Rebuild Bihar person cases "
+
+    def handle(self, *args,**kwargs):
+        person_config = _get_config_by_id('static-icds-cas-static-person_cases_v3')
+        table_name = get_table_name('icds-cas', 'static-person_cases_v3')
+        query = """select supervisor, doc_id from "{}" where state_id='f9b47ea2ee2d8a02acddeeb491d3e175' order by supervisor, doc_id""".format(table_name)
```

Comment:
given that there's been some discussion around ordering, and that it's important, it might make sense to note that in a comment so it doesn't get lost